aries_cloudagent/config/argparse.py

```diff
@@ -1694,6 +1705,10 @@ def get_settings(self, args: Namespace) -> dict:
             settings["wallet.type"] = args.wallet_type
         if args.wallet_key_derivation_method:
             settings["wallet.key_derivation_method"] = args.wallet_key_derivation_method
+        if args.wallet_key_derivation_method:
+            settings["wallet.rekey_derivation_method"] = (
+                args.wallet_rekey_derivation_method
```

Comment:
Why does it use the `wallet_rekey_derivation_method` value when `wallet_key_derivation_method` is declared?

Reply:
Yes. I think this was a careless copy/paste error. Good spot. The brackets are just for the line length.